Containerfile

```diff
@@ -0,0 +1,12 @@
+FROM fedora:34
+
+RUN dnf upgrade -y && \
+    dnf install -y findutils-1:4.8.0-2.fc34 \
+                   linkchecker-9.4.0-12.20191005.d13b3f5.fc34 \
+                   make-1:4.3-5.fc34 \
+                   ruby-3.0.2-149.fc34
+
+RUN gem install --pre asciidoctor:2.0.16 \
+                      asciidoctor-pdf:1.6.0
```

Comment:
I pinned packages to exact versions to allow others to rebuild to same "exact" container image. Removed.

I've went this way because of the instructions in `guides/README.md`. I've added a commit to correct the pages names accordingly.